docs/source/trainer.rst

```diff
@@ -1718,3 +1718,14 @@ The metrics sent to the progress bar.
 
     progress_bar_metrics = trainer.progress_bar_metrics
     assert progress_bar_metrics['a_val'] == 2
+
+
+Benchmark with vanilla PyTorch
+------------------------------
+
+We have set regular benchmarking against PyTorch vanilla training loop on with RNN and simple MNIST classifier as per of out CI.
```

Comment:
Can you add the date there ?

Reply:
any suggestion if put in the title or just as a footnote to the text?

Reply:
added as fig captions